sda-commons-web-autoconfigure/src/test/resources/application.properties

```diff
@@ -1,3 +1,5 @@
 management.server.port=0
 wiremock.server.port=0
-spring.zipkin.baseUrl=http://localhost:${wiremock.server.port}/zipkin
\ No newline at end of file
+spring.zipkin.baseUrl=http://localhost:${wiremock.server.port}/zipkin
+
+server.tomcat.basedir=static
```

Comment:
Overwriting the property from `defaults.properties` here will imho create a misleading test case.

I tried locally with removing this:

```suggestion
```

and changing the assertion above:

```diff
- assertThat(testApp.getBaseDir()).isEqualTo("static");
+ assertThat(testApp.getBaseDir()).endsWith("/tomcat");
```

the tests are green then.

But anyways, service developers still have to create `/tmp/tomcat` in addition to `/static` in their container, don't they?